loadtest/user/userentity/websocket.go

```diff
@@ -18,6 +18,22 @@ const (
 	maxWebsocketFails             = 7
 )
 
+func (ue *UserEntity) handlePostEvent(ev *model.WebSocketEvent) error {
+	var post *model.Post
+	if err := json.Unmarshal([]byte(ev.Data["post"].(string)), &post); err != nil {
```

Comment:
I would be extra-cautious and do a check of whether "post" exists or not and whether it is castable to string or not. There have been cluster bugs where map properties don't get serialized properly.

Reply:
I could argue that panicking is our best chance of catching the bug. A rare error here might easily get unnoticed in the huge stream of logs that this tool will generate.

Reply:
Hmm .. but do we want to crash the entire load test tool just to catch a server bug ? I would think we would just log the error and move on. Panicking is usually an irrecoverable condition for a process, and I dont think thats the case here.

We can come up with ways of making errors more prominent later by using metrics or other log analysis tools, but I think thats an orthogonal concern.

Reply:
Of course and I guess that's the point I was trying to raise with my slightly *trolling* argument. We'll need to be extra careful about errors. I was trying to give a different point of view since we discovered several bugs in the main server code thanks to unexpected panics.